Packs/SentinelOne/Integrations/SentinelOne-V2/README.md

```diff
@@ -288,7 +289,7 @@ Applies a mitigation action to a group of threats that match the specified input
 | --- | --- | --- |
 | SentinelOne.Threat.ID | String | The threat ID. | 
 | SentinelOne.Threat.Mitigated | Boolean | Whether the threat was successfully mitigated. | 
-| SentinelOne.Threat.Mitigation.Action | Number | Number of threats affected. | 
+| SentinelOne.Threat.Mitigation.Action | String | The mitigation action performed. | 
```

Comment:
@chloerongier Will this break backwards compatibility for playbooks, or is this just correcting the type to match what is currently available?

Reply:
Hello, This is just correcting the type to match what is currently done by the code!